### ISSUE TITLE
Incorect breadcrumb redirection from static/cms pages for non-default stores

### ISSUE THREAD

**Issue by ArturDivante** [DivanteLtd/vue-storefront#3631]
## Current behavior
Homepage breadcrumb tries to redirect to default store instead of specific. As a result - empty page is displayed.

## Expected behavior
Homepage breadcrumb should navigate correctly to specific store homepage.

## Steps to reproduce the issue
1. With multistore enabled, switch to non-default langiage.
2. Open one of the static or CMS pages from the footer (i.e. http://localhost:3000/it/customer-service)
3. Find homepage breadcrumb and click it

## Note - Pls don't skip it while reading ;)
Similar problem (which propably could be fixed with this1) applies to legal note and privacy policy links from the very bottom of the page. Both of them are hyperlinked to the default store.

## Repository
Latest develop

## Can you handle fixing this bug by yourself?

- [ ] YES
- [x] NO

## Which [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html) state this refers to? Info for developer.
Pick one option.

- [ ] This is a bug report for test version on https://test.storefrontcloud.io - In this case Developer should create branch from `develop` branch and create Pull Request `2. Feature / Improvement` back to `develop`.
- [x] This is a bug report for current Release Candidate version on https://next.storefrontcloud.io - In this case Developer should create branch from `release` branch and create Pull Request `3. Stabilisation fix` back to `release`.
- [ ] This is a bug report for current Stable version on https://demo.storefrontcloud.io and should be placed in next stable version hotfix - In this case Developer should create branch from `hotfix` or `master` branch and create Pull Request `4. Hotfix` back to `hotfix`.

## Environment details
- Browser: Latest Chrome
- OS: Ubuntu, Win10
- Code Version: Develop

**Comment by andrzejewsky** [comment]
@ArturDivante 
> Both of them are hyperlinked to the default store

Yeah because we don't have translations for them, it was introduced here https://github.com/DivanteLtd/vue-storefront/pull/3592

**Comment by ArturDivante** [comment]
@andrzejewsky 
Ok. I have no problem with them being linked there (for now). But with the fact, this redirection is not working, and throws user to empty page when clicked from de or it store ;)

**Comment by andrzejewsky** [comment]
@ArturDivante yeah agree with breadcrumb, that needs to be fixed.

**Comment by grimasod** [comment]
I'll fix this, as part of PR #3267. Ready today

**Comment by patzick** [comment]
@grimasod we should take a closer look into #3267 as #3633 was aiming for fix here.
I believe we could try to add this to RC2 after CR :)

**Comment by grimasod** [comment]
@patzick Ok, I can look at it again. When I checked #3633 last week, it didn't resolve the same issues, but I see it has some of the same functionality now.

One difference is that #3633 has most code in the theme breadcrumb component, whereas I used the breadcrumbs module much more in #3267

> I think we should get rid of breadcrumbs module eventually because it's not necessary

I found the breadcrumbs module is necessary to avoid calculating the categories twice (once for breadcrumbs and once to retrieve the categories). If you've sure the breadcrumbs module should be deprecated, then I can try to find another solution
